### ISSUE TITLE
Prevent adding `px` for variables that have unquoted (numeric) values

### ISSUE THREAD

**Issue by 7iomka** [mertasan/tailwindcss-variables#23]
### Versions:
- Package Version: latest
- Tailwindcss Version: latest

### Steps To Reproduce:

```js
variables: {
      DEFAULT: {
        // body
        'body-color': '#000',
        'body-bg': '#fff',
        'body-font-family': 'var(--font-primary)',
        'body-font-size': '1rem',
        'body-font-weight': 400,
        'body-line-height': 1.5,
      },
```

produces
```css
    --body-color: #000;
    --body-bg: #fff;
    --body-font-family: var(--font-primary);
    --body-font-size: 1rem;
    --body-font-weight: 400px;
    --body-line-height: 1.5px;
```

`400px`, `1.5px` - why?
Hot to fix that?

**Comment by mertasan** [comment]
@7iomka The test is passing. 4c9c0e19a6553c44558d56a697361e5b94ef50af

**Comment by 7iomka** [comment]
Maybe something happens with my config?
```js
// const defaultTheme = require("tailwindcss/defaultTheme");
// const plugin = require('tailwindcss/plugin');
const { corePlugins } = require('tailwindcss/lib/corePlugins');

const bootstrapGridPlugin = require('tailwind-bootstrap-grid');
const cssVariablesPlugin = require('@mertasan/tailwindcss-variables');

const breakpoints = {
  xs: '360px',
  xsm: '375px',
  xsl: '414px',
  sm: '576px',
  smd: '640px',
  md: '768px',
  lg: '992px',
  xl: '1200px',
  lp: '1366px',
  xxl: '1400px',
};

const containerMaxWidths = {
  sm: '540px',
  md: '720px',
  lg: '960px',
  xl: '1140px',
  xxl: '1320px',
};

const spacerBase = 2; // 2px
const remBaseline = 16; // 16px
const remBaselinePercentage = remBaseline / 16;

const round = (value, precision = 4) => {
  const power = Math.pow(10, precision);
  return Math.round(value * power + Number.EPSILON * power) / power;
};

const rem = (pxValue) => round(pxValue / remBaseline);

const spacers = {
  0: 0,
};

for (let index = 1; index <= 200; index++) {
  spacers[`${index}`] = `${rem(index * spacerBase)}rem`;
}

module.exports = {
  // container: {
  //   center: true,
  //   padding: {
  //     DEFAULT: '0.9375rem', // 15px
  //     md: '1rem', // 16px
  //     xl: '3.4375rem', // 55px
  //   },
  // },
  darkMode: 'class',
  theme: {
    // fontSize: {
    //   custom: 'var(--font-size-custom)',
    // },
    colors: ({ colors }) => ({
      primary: 'var(--c-primary)',
      secondary: 'var(--c-secondary)',
      success: 'var(--c-success)',
      danger: 'var(--c-danger)',
      gray: {
        100: 'var(--c-gray-100)',
        200: 'var(--c-gray-200)',
        300: 'var(--c-gray-300)',
        400: 'var(--c-gray-400)',
        500: 'var(--c-gray-500)',
        600: 'var(--c-gray-700)',
        700: 'var(--c-gray-800)',
        800: 'var(--c-gray-700)',
        900: 'var(--c-gray-900)',
      },
      inherit: colors.inherit,
      current: colors.current,
      transparent: colors.transparent,
      black: colors.black,
      white: colors.white,
      inherit: colors.inherit,
      current: colors.current,
      transparent: colors.transparent,
      black: colors.black,
      white: colors.white,
    }),
    // Configure variables for default mode
    variables: {
      DEFAULT: {
        // theme
        primary: '#ffda7c',
        secondary: '#fe7375',
        success: '#58dd4d',
        danger: '#fe7375',
        // grays
        gray: {
          100: '#F8F8F8',
          200: '#E0E0E0',
          300: '#C8C8C8',
          400: '#888888',
          500: '#707070',
          600: '#505050',
          700: '#383838',
          800: '#282828',
          900: '#101010',
        },
        // typograpy
        'font-primary': 'Acrom, Helvetica, sans-serif',
        'font-serif': 'georgia, "Times New Roman", times, serif',
        // body
        'body-color': '#000',
        'body-bg': '#fff',
        'body-font-family': 'var(--font-primary)',
        'body-font-size': '1rem',
        'body-font-weight': 'normal',
        'body-line-height': '1.5em', // currently but there if use without unit (em)

        // 'font-size-custom': '1rem',
      },
    },
    // Configure variables for dark mode (optional)
    darkVariables: {
      DEFAULT: {
        primary: '#ffda7c',
        secondary: '#fe7375',
        success: '#58dd4d',
        danger: '#fe7375',
        gray: {
          100: '#1b1b29',
          200: '#2b2b40',
          300: '#323248',
          400: '#474761',
          500: '#565674',
          600: '#6d6d80',
          700: '#92929f',
          800: '#cdcdde',
          900: '#ffffff',
        },
        // 'font-size-custom': '1rem',
      },
    },
    spacing: spacers,
    extend: {
      screens: breakpoints,
      fontFamily: {
        sans: ['var(--font-primary)'], // 'Acrom', 'Helvetica', 'sans-serif'
        serif: ['var(--font-serif)'], // 'georgia', 'Times New Roman', 'times', 'serif'
      },
      maxWidth: {
        // Add .max-width-container-xxx
        ...Object.keys(containerMaxWidths).reduce(
          (prev, curr) => ({ ...prev, [`container-${curr}`]: containerMaxWidths[curr] }),
          {},
        ),
      },
    },
  },
  plugins: [
    cssVariablesPlugin({
      darkToRoot: true,
      darkSelector: '[data-mode="dark"]',
      colorVariables: true,
    }),
    bootstrapGridPlugin({
      containerMaxWidths,
      gridGutters: spacers,
    }),
  ],
  corePlugins: {
    ...corePlugins,
    container: false, // use bootstrap grid implementation instead
  },
};

```

**Comment by 7iomka** [comment]
@mertasan Issue actual if values is unquoted (numbers in this case)

**Comment by mertasan** [comment]
Thanks for this! 

fixed in v2.0.1 and v1.3.4.
